clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -234,6 +235,17 @@
      */
     static final String LEAVE_GROUP_ON_CLOSE_CONFIG = "internal.leave.group.on.close";
 
+    /** <code>isolation.level</code> */
+    public static final String ISOLATION_LEVEL_CONFIG = "isolation.level";
+    public static final String ISOLATION_LEVEL_DOC = "<p>Controls how to read messages written transactionally. If set to <code>READ_COMMITTED</code>, consumer.poll() will only return" +
```

Comment:
I changed everything to lower case, but retained the 'read' prefix, since we are trying to use the database terminology with 'isolation.level'.